ramls/schema/request-policy-id.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Result of applying circulation rules to determine a request policy",
+  "description": "Describes the outcome of applying circulation rules to determine a request policy",
+  "type": "object",
+  "properties": {
+    "loanPolicyId": {
```

I think this is meant to be `requestPolicyId`
```suggestion
    "requestPolicyId": {
```

---

Corrected in https://github.com/folio-org/mod-circulation/pull/190/commits/91f57fec6f3c9ef6886ce4c49014a49fa9da21c9